From 4a3d34e60b868473460c30e7d2c25d95bf5c3931 Mon Sep 17 00:00:00 2001
From: Andy Wilkinson <awilkinson@pivotal.io>
Date: Fri, 23 Oct 2015 16:49:50 +0100
Subject: [PATCH] Remove compile warning workaround that adds unwanted
 dependency

---
 build.gradle | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build.gradle b/build.gradle
index 2532385..c5a1a86 100644
--- a/build.gradle
+++ b/build.gradle
@@ -77,7 +77,6 @@ dependencies {
 
 	// GemFire
 	compile("com.gemstone.gemfire:gemfire:$gemfireVersion")
-	optional("com.google.code.findbugs:annotations:2.0.2")
 	runtime("antlr:antlr:$antlrVersion")
 
 	compile "org.aspectj:aspectjweaver:$aspectjVersion"
-- 
2.2.1

