From 6ccda259db97b86ca8439439fcb8df488de26789 Mon Sep 17 00:00:00 2001
From: Andy Wilkinson <awilkinson@pivotal.io>
Date: Wed, 1 Feb 2017 14:05:54 +0000
Subject: [PATCH] Upgrade Spring IO plugin so it's Gradle 3 compatible

---
 build.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.gradle b/build.gradle
index c1ec57df..976e231c 100644
--- a/build.gradle
+++ b/build.gradle
@@ -7,7 +7,7 @@ buildscript {
         classpath 'me.champeau.gradle:gradle-javadoc-hotfix-plugin:0.1'
         classpath('org.asciidoctor:asciidoctor-gradle-plugin:0.7.0')
         classpath('org.asciidoctor:asciidoctor-java-integration:0.1.4.preview.1')
-        classpath 'io.spring.gradle:spring-io-plugin:0.0.4.RELEASE'
+        classpath 'io.spring.gradle:spring-io-plugin:0.0.6.RELEASE'
     }
 }
 
-- 
2.11.0

