From 78e35d96074be057870e514df18ece898e455681 Mon Sep 17 00:00:00 2001
From: Andy Wilkinson <awilkinson@pivotal.io>
Date: Wed, 16 Jan 2019 14:44:21 -0500
Subject: [PATCH] Align mockwebserver's version with Platform's version of OK
 HTTP

---
 gradle/dependency-management.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gradle/dependency-management.gradle b/gradle/dependency-management.gradle
index 6c9aa3f48..c9756331a 100644
--- a/gradle/dependency-management.gradle
+++ b/gradle/dependency-management.gradle
@@ -6,7 +6,7 @@ dependencyManagement {
 	}
 	dependencies {
 		dependency 'cglib:cglib-nodep:3.2.10'
-		dependency 'com.squareup.okhttp3:mockwebserver:3.12.1'
+		dependency 'com.squareup.okhttp3:mockwebserver:3.10.0'
 		dependency 'opensymphony:sitemesh:2.4.2'
 		dependency 'org.gebish:geb-spock:0.10.0'
 		dependency 'org.jasig.cas:cas-server-webapp:4.2.7'
-- 
2.18.0

