From 969c01e492c110c4d4fdaa634dd77b8638b07d67 Mon Sep 17 00:00:00 2001
From: Andy Wilkinson <awilkinson@pivotal.io>
Date: Thu, 28 Apr 2016 13:28:41 +0100
Subject: [PATCH] Disable IO Platform tests in infrastructure-neo4j4 when using
 SDNeo4j4

---
 build.gradle | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/build.gradle b/build.gradle
index ed26c4b..7391f97 100644
--- a/build.gradle
+++ b/build.gradle
@@ -377,6 +377,15 @@ project('spring-batch-infrastructure-neo4j4') {
 		testCompile "org.mockito:mockito-core:$mockitoVersion"
 		testCompile "junit:junit:${junitVersion}"
 	}
+
+	if (project.hasProperty('platformVersion')) {
+		def dataNeo4jVersion = dependencyManagement.springIoTestRuntime.managedVersions['org.springframework.data:spring-data-neo4j']
+		if (dataNeo4jVersion?.startsWith('3.')) {
+			tasks.withType(Test).matching { it.name =~ "springIoJdk.Test" }.all {
+				enabled false
+			}
+		}
+	}
 }
 
 project('spring-batch-core-tests') {
-- 
2.6.4

