From 6063b1b2a7d276c131f3565bf7f63db9f935371a Mon Sep 17 00:00:00 2001
From: Andy Wilkinson <awilkinson@pivotal.io>
Date: Mon, 17 Oct 2016 10:27:41 +0100
Subject: [PATCH] Upgrade Jacoco to avoid problems caused by Java 8 default
 methods

---
 build.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.gradle b/build.gradle
index ed26c4b..848a949 100644
--- a/build.gradle
+++ b/build.gradle
@@ -107,7 +107,7 @@ configure(subprojects - project(":spring-build-src")) { subproject ->
 	apply plugin: 'merge'
 
 	jacoco {
-		toolVersion = "0.7.0.201403182114"
+		toolVersion = "0.7.7.201606060606"
 	}
 
     compileJava {
-- 
2.9.0

