From ee1ca14e86f43830ad8fc5c6ef3bc563590f6bef Mon Sep 17 00:00:00 2001
From: Andy Wilkinson <awilkinson@pivotal.io>
Date: Wed, 15 Feb 2017 16:34:01 +0000
Subject: [PATCH] Update the javacc download URL to use javacc.org

---
 core/javacc.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/javacc.gradle b/core/javacc.gradle
index 15a83ec4..81439d09 100644
--- a/core/javacc.gradle
+++ b/core/javacc.gradle
@@ -2,7 +2,7 @@
 task downloadJavacc {
     ext.file = "javacc-5.0.tar.gz"
     ext.dest = file("$buildDir/javacc/$file")
-    ext.url = "https://java.net/downloads/javacc/$file"
+    ext.url = "http://javacc.org/downloads/$file"
 
     outputs.file dest
 
-- 
2.11.0

