From e9c3845aed3e8e19bbcf8a9eca6c8ef256c30e9f Mon Sep 17 00:00:00 2001
From: Andy Wilkinson <awilkinson@pivotal.io>
Date: Mon, 5 Mar 2018 10:26:00 +0000
Subject: [PATCH] Patch project for IO Platform compatibility

---
 build.gradle | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/build.gradle b/build.gradle
index 32b21096..d13f5469 100644
--- a/build.gradle
+++ b/build.gradle
@@ -111,6 +111,9 @@ configure(subprojects - project(":spring-build-src")) { subproject ->
             imports {
                 mavenBom "io.spring.platform:platform-bom:${springIoVersion}"
             }
+            dependencies {
+                dependency "org.thymeleaf:thymeleaf-spring4:${thymeleaf2Version}"
+            }
         }
     }
 
@@ -284,6 +287,7 @@ project("spring-social-security") {
         compile("org.springframework.security:spring-security-web:$springSecurityVersion")
         compile("javax.servlet:javax.servlet-api:$servletApiVersion", provided)
     }
+    ext['mockito.version'] = mockitoVersion
 }
 
 configure(rootProject) {
-- 
2.14.1

